src/components/shop/shop.js

```diff
@@ -110,3 +112,9 @@ export class Shop extends Screen {
         );
     }
 }
+
+export const shopOverlay = (screen, shopKey) => {
```

Comment:
Interesting adding the stats export - should save agencies having to look it up. 

Possibly should be verb here rather than noun? "addShopOverlay" ? "openShop"?